diff --git a/DOL-terms.rdf b/DOL-terms.rdf
index de42e38..6f03775 100644
--- a/DOL-terms.rdf
+++ b/DOL-terms.rdf
@@ -1022,7 +1022,7 @@ SubPropertyChain: isProfileOf o supportsSerialization</rdfs:comment>
 
     <owl:Class rdf:about="&dol;AbstractSyntax">
         <rdfs:label xml:lang="en">abstract syntax</rdfs:label>
-        <rdfs:subClassOf rdf:resource="&dol;LinguisticEntity"/>
+        <rdfs:subClassOf rdf:resource="&dol;Language"/>
         <skos:note xml:lang="en">An abstract syntax can be specified as a MOF metamodel. Then abstract abstract syntax documents can be represented as  XMI documents.</skos:note>
         <skos:definition xml:lang="en">term language for representing documents in a machine-processable way</skos:definition>
     </owl:Class>
